CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -pedantic -Wall -Wextra")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -pedantic -Wall -Wextra")
```

Comment:
Actually it turns out someone codemodded CMake's own repo to use `string(APPEND)` where applicable: https://github.com/Kitware/CMake/commit/5d0d980d9949daf596e10715d686adc95c1c232b